notebooks/Notebook2.md

```diff
@@ -2,33 +2,56 @@
+New PassingPlayer(playerOrNum).setYards(numYards).setINCs(outcome).setTDs(outcome).setINTs(outcome).updatePlayer();
+This was useful because I was able to handle inputs that are the same across different methods. ([code here](https://github.com/hmc-cs111-spring2023/BBeckerArtifact/tree/main/src/Player))
+
+Finally I built a parser that takes each part of the input language and utilizes the Game class add function to add a player into the data structure. This involved building code to determine what type of playtype each input from the user was. These functions are my is…() functions. They return a boolean value if the input matches a possible input string. This could be further simplified by using another type of parser but for now It works great. Next I worked on function that are designed to add the player to the game and return the game to the parser these function utilize the add function from the game class to add a new player to the data structure or if the player already exists update the stats by merging the players stats that were just created with the old ones. The individualized settlers for each player type came in really nice for this aspect. Finally I worked on some functions that would transform strings to numbers and parsing out yardage from a string. For example, if the user inputs 22y representing 22 yards the parser needs to transform that into just an integer number 22. ([code here](https://github.com/hmc-cs111-spring2023/BBeckerArtifact/blob/main/src/Parser/Parser.java))
+
 
 ## Questions
 
 **What is the most pressing issue for your project? What design decision do
 you need to make, what implementation issue are you trying to solve, or how
 are you evaluating your design and implementation?**
 
+The most pressing issue right now is to get my repo set up in a way that allows for me to build a physical interface for the user to input the language. I think this is the best way for the user to use my dsl and it's better than using a command line input. Also I will begin to work on error handling for the language next week and I think that this will best be presented to the user with such an interface. 
+
+
 **What questions do you have for your critique partners? How can they best help
 you?**
 
+If anyone sees any possible ways of simplifaction in the code I would love to know?
+
+
 **How much time did you spend on the project this week? If you're working in a
 team, how did you share the work?**
 
+This week I spent roughly 8 hours on the project. 
+
+
 **Compared to what you wrote in your contract about what you want to get out of this
 project, how did this week go?**
+
+This week went well. I finished building the bridge from front to back and I am able to test almost all of the components of my language this way. I am way ahead of schedule and I plan to work on language arrow handling as well as starting to work on the API portion of the project. 
```

Comment:
Great progress overall, I liked that you explored the idea of a GUI because you've gotten ahead on your project and I look forward to seeing how the API portion of your project comes along!